src/Compilers/CSharp/Portable/Symbols/Wrapped/WrappedParameterSymbol.cs

```diff
@@ -154,6 +154,7 @@ public override string GetDocumentationCommentXml(CultureInfo? preferredCulture
 
         internal sealed override ScopedKind EffectiveScope => _underlyingParameter.EffectiveScope;
 
+        // TODO: sealed?
```

Comment:
TODO

Reply:
I'm going to address this in the base branch